### PR TITLE
Mobile-Desktop-popup

### PR DESCRIPTION
**Pull request by 1GORDON**
## We achieved the following in this milestone ##

- Created an object to store properties
- Added mobile and desktop popup screens
- Added blur background on the desktop popup

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,414 +1,355 @@
...
+              alt="union"
+            />
+          </a>
+          <ul>
+            <li><a href="#">Portolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </nav>
+      </header>
```
Comment (DanteAlonsoHT) [inline index.html:44]: - [x] You can use `max-width` CSS attribute of another responsive design technique to make it better for any screen size.
- [x] Check the red lines to check what you should improve (not all if you want) but I strongly recommend you trying to make your design more similar to the Figma design you selected.
- [x] Add the right background when you are seeing your popup active.

Your mobile design | Figma mobile design

image

File: index.html [hunk 2]
```diff
@@ -1,414 +1,355 @@
...
+                </div>
             </div>
-        </section>
-    </section>
-</section>
+        </div>
+    </div>
+
+    <!-- project section -->
+    <section class="projects"></section>
```
Comment (DanteAlonsoHT) [inline index.html:138]: - [x] Check the blue lines to see what you can improve in your desktop popup design.
- [x] Add the right background.
- [x] Your text changes a lot when I try to resize the desktop version, kindly fix it due to sometimes part of your text disappears when I resize the tab.

Your desktop design

image

Figma desktop design

image

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

Hi @1GORDON, 

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] - Create "css" folder and put all .css files inside this folder.
- [ ] - Create "js" folder and put all .js files inside this folder.
- [ ] - Create "img" folder and put all .png, .jpeg, .jpg, .svg files (all images and vectors) inside this folder.

It can help you to maintain your repository more ordered and clear 😉 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (1GORDON) [reply index.html:138]: Thank you for the review. let me make the necessary changes

File: index.html [hunk 3]
```diff
@@ -1,414 +1,358 @@
...
-                    <li><img src="images/counter-1.png" alt="counter1"></li>
-                    <li><p>Back End Dev</p></li>
-                    <li><img src="images/counter-2.png" alt="counter2"></li>
-                    <li><p>2015</p></li>
-                </ul>
-            </div>
-            <p class="content">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <br>
-            <div class="buttons">
-                <ul class="list">
```
Comment (SajjadAhmad14) [inline index.html:77]: The project section is hidden on medium and small viewports.  So I am unable to review the pop-up details. Kindly see the below screenshot for reference.
missing

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Godanah Tinyefuza, 
 ### Thank you for implementing changes suggested previously:ok_hand:

## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 4]
```diff
@@ -1,414 +1,358 @@
...
+                </div>
             </div>
-        </section>
-    </section>
-</section>
+        </div>
+    </div>
+
+    <!-- project section -->
+    <section class="projects"></section>
```
Comment (DanteAlonsoHT) [inline index.html:138]: - [ ] [OPTIONAL] Kindly add more padding for your tech stack labels or fix the content according to the size of your tech stack name. 😉 
- [ ] [OPTIONAL] Make it responsive, you can use min-width CSS attribute to fix it 👍 (check the second image below) 

image

image

**Review by DanteAlonsoHT — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @1GORDON, 

### Optional Changes Required
_Check in the comments below_

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**